config/operator/global/webhook.template.yaml

```diff
@@ -3,6 +3,25 @@ kind: ValidatingWebhookConfiguration
 metadata:
   name: elastic-webhook.k8s.elastic.co
 webhooks:
+  - clientConfig:
+      caBundle: Cg==
+      service:
+        name: elastic-webhook-server
+        namespace: <NAMESPACE>
+        # this is the path controller-runtime automatically generates
+        path: /validate-elasticsearch-k8s-elastic-co-v1-elasticsearch
+    failurePolicy: Ignore
+    name: elastic-es-validation.k8s.elastic.co
+    rules:
+      - apiGroups:
+          - elasticsearch.k8s.elastic.co
+        apiVersions:
+          - v1
```

Comment:
I'm not sure it worth keeping the reference to v1beta1 in the webhook, webhook was not installed/used in 1.0.0.beta.

Reply:
We discussed with the team how it probably makes sense to have it, otherwise users could potentially deploy a `v1beta1` resource, stored as `v1`, with no webhook executed. Which feels a bit like an inconsistent escape hatch.

I need to do some testing though, I think right now the v1beta1 webhook does not work.